### ISSUE TITLE
upgrade to pyyaml 5.3+

### ISSUE THREAD

**Issue by glyph** [aws/aws-cli#5122]
pyyaml 5.3.1 is out.

**Comment by nateprewitt** [comment]
Thanks for letting us know, @Glyph! We currently allow users to upgrade to anything <5.4 on Python 2.7, 3.5+. We're taking a look at any other pin updates needed to address this.

**Comment by glyph** [comment]
Thanks @nateprewitt !

I guess this specific issue should be closed, then?  I swear the pin was set lower when I filed this.

Also, would you consider some kind of automation for moving these pins?  I'm constantly blocked on automatic upgrades because aws-cli happens to be in our dev dependencies.

**Comment by glyph** [comment]
Oh, I see, I was scanning for the pin, and saw the 3.4 conditional in setup.py.

**Comment by nateprewitt** [comment]
Ok, it doesn't look like there's any other tweaks we can make for 3.4 here, so I think this is ready to close.

I'll bring up adding something like Dependabot with the team so you're not having to ping us any time there's a release. We can definitely open another issue if there's interest in tracking progress/discussion on that. Thanks again, @glyph!

**Comment by glyph** [comment]
Thanks @nateprewitt !  I'd be interested to hear if you can get dependabot (or requires.io or pyup.io) to honor shifting inequality pins like this; I know they'll move `==` in a requirements.txt for you (that's how I get notified of auto-updates on my own projects) but I haven't seen them move `<` before.
